docs/plans/2026-05-20-credential-stabilize-sweep-plan.md

```
@@ -0,0 +1,2522 @@
+last-reviewed: 2026-05-20
+```
+
+- [ ] **Step 3: Flip credential-runtime README frontmatter**
+
+Same change in `crates/credential-runtime/README.md`.
+
+- [ ] **Step 4: Update MATURITY.md**
+
+In `docs/MATURITY.md`, locate the `nebula-credential` row and change the status column from `frontier` to `stable`. Add a note (per existing format):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)